internal/services/compute/migration/legacy_vmss.go

```diff
@@ -664,6 +664,11 @@ func (LegacyVMSSV0ToV1) Schema() map[string]*pluginsdk.Schema {
 						Optional: true,
 					},
 
+					"automatic_upgrade_enabled": {
+						Type:     pluginsdk.TypeBool,
+						Optional: true,
+					},
```

Comment:
this is a state migration of a given point in time - as such this needs to be reverted:

```suggestion
```